Epsilon.Abstractions/LearningDomainOutcomeResult.cs

```diff
@@ -2,8 +2,5 @@ namespace Epsilon.Abstractions;
 
 public record LearningDomainOutcomeResult(
     LearningDomainOutcome Outcome,
-    double Grade,
-    DateTime SubmittedAt,
-    string Assignment,
-    Uri AssignmentUrl
+    double? Grade
```

Comment:
Can grade be null? Shouldn't be null imo, but this is probably because of the kpi matrix?

Reply:
Correct, When not graded, value will be null. And is needed for KPI matrix

Reply:
Should probably not be an outcome result then maybe? As this is usually where grades are set. Because this is for individual outcomes

Reply:
True when following that logic the name is incorrect